ftw_tools/torchgeo/trainers.py

```diff
@@ -240,6 +266,22 @@ def configure_models(self) -> None:
         if patch_weights:
             self.transfer_weights(self.model, backbone)
 
+    def _log_per_class(self, metrics_dict, split: str):
```

Comment:
Can use ClasswiseWrapper here instead if you want metrics recorded per class also btw https://lightning.ai/docs/torchmetrics/stable/wrappers/classwise_wrapper.html

Reply:
I did this on purpose because I wanted to see the class names in the output / torchmetrics (and lightning self.log...) do a _little_ too much hidden magic for me.